### PR TITLE
Create newcomers redirection file

### LINKED ISSUE #1890: Create a redirect file so that the "Being a part of the HFLA team" wiki article can be reached from the hfla website
### Overview
The ["Being a Part of the Hack For LA Team" wiki](https://github.com/hackforla/website/wiki/Being-a-Part-of-the-Hack-For-LA-Team) is a good document for newcomers to get an idea of how to get on-boarded. However, finding the wiki can be difficult, and people may forget the exact url to reach it. 

During the developer meeting, we discussed that making a redirect file may be the easiest way to circumvent the issue. It'll provide an easy, memorable url for people to immediately access the wiki. A good redirect url could be hackforla.org/newcomers or something similar.

### Action Items
- [x] Create a new redirect file that redirects users from https://www.hackforla.org/newcomers (or something similar) to the ["Being a Part of the Hack For LA Team" wiki](https://github.com/hackforla/website/wiki/Being-a-Part-of-the-Hack-For-LA-Team) page.

### Resources/Instructions
[Redirections according to the site architecture](https://github.com/hackforla/website/wiki/Hack-for-LA's-Site-Architecture#redirections)

Example:
Going to https://www.hackforla.org/donate/ redirects the user to https://www.codeforamerica.org/donate/

### PR DESCRIPTION
**Pull request by vickiychung**
Fixes #1890 

### What changes did you make and why did you make them ?

  - Created `newcomers.md` in the `redirections` directory

### CODE REVIEW

**Review by Zak234 — DISMISSED** [review]
Everything seems to be working. Good job!

**Review by macho-catt — CHANGES_REQUESTED** [review]
Looks good and the redirection works for me as well! A couple of things (very minor typos/grammar), newcomer is one word, so the title should be Newcomers, not New Comers. Also, in the comment, 'redirections' should be 'redirects.'
Otherwise, it's good!

File: redirections/newcomers.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+# https://www.hackforla.org/newcomers redirections to -> https://github.com/hackforla/website/wiki/Being-a-Part-of-the-Hack-For-LA-Team
+layout: redirect
+title: New Comers
```
Comment (macho-catt) [inline redirections/newcomers.md]: should be Newcomers (one word)

File: redirections/newcomers.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+# https://www.hackforla.org/newcomers redirections to -> https://github.com/hackforla/website/wiki/Being-a-Part-of-the-Hack-For-LA-Team
```
Comment (macho-catt) [inline redirections/newcomers.md]: 'redirections' here should be 'redirects' instead

File: redirections/newcomers.md (hunk 1 above)
Comment (vickiychung) [reply redirections/newcomers.md]: Updated! Thanks

File: redirections/newcomers.md (hunk 2 above)
Comment (vickiychung) [reply redirections/newcomers.md]: Updated

**Review by macho-catt — APPROVED** [review]
lgtm!

**Pull request merged**